deploy/helm/kubecf/BUILD.bazel

```diff
@@ -75,3 +80,13 @@ helm_version(
     name = "version",
     chart = ":kubecf",
 )
+
+image_list(
+    name = "image_list",
+    chart = ":kubecf",
+)
+
+print_files(
```

Comment:
Add some comments please for what these do.

Reply:
Fixed with https://github.com/cloudfoundry-incubator/kubecf/pull/571/commits/5fcab07449d5553739b0266bce81a105f48fa201.